HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -210,6 +210,12 @@ def customiseForOffline(process):
 
     return process
 
+def customizeForRecoPixelVertexing(process):
```

Comment:
Since this will eventually disappear, it might be clearer to use the usual naming convention, e.g. `customizeHLTfor41058`. I don't think it's worth doing it now, unless this PR will have to be updated. I might do this renaming the next time I touch this file, though, if that's okay.